Networking/Networking/Model/Refund/Refund.swift

```diff
@@ -130,14 +130,13 @@ extension Refund: Comparable {
     public static func == (lhs: Refund, rhs: Refund) -> Bool {
         return lhs.refundID == rhs.refundID &&
             lhs.orderID == rhs.orderID &&
+            lhs.siteID == rhs.siteID &&
             lhs.dateCreated == rhs.dateCreated &&
             lhs.amount == rhs.amount &&
             lhs.reason == rhs.reason &&
-            lhs.refundedByUserID == rhs.refundedByUserID &&
-            lhs.isAutomatedRefund == rhs.isAutomatedRefund &&
-            (!lhs.items.isEmpty && !rhs.items.isEmpty) ?
-        lhs.items.count == rhs.items.count &&
-        lhs.items.sorted() == rhs.items.sorted() : true
+            lhs.byUserID == rhs.byUserID &&
+            lhs.isAutomated == rhs.isAutomated &&
+            lhs.items.sorted() == rhs.items.sorted()
```

Comment:
Do you think the diffs in the Networking layer (just this file) in this PR could be moved to the Networking PR (https://github.com/woocommerce/woocommerce-ios/pull/1355)? so that this PR can also be merged independently of the Networking PR (after updating the PR target branch)

Reply:
Yep, I can do that. I changed Networking code in the wrong branch 🤦‍♀